src/main/java/org/symphonyoss/integration/jira/api/JiraApiResource.java

```diff
@@ -121,6 +126,30 @@ public ResponseEntity searchAssignableUsers(@RequestParam String issueKey,
   }
 
   /**
+   * Get information of a specific issue.
+   * @param issueKey Issue identifier
+   * @return List of potential assigneers users or 400 Bad Request - Returned if no issue key
```

Comment:
It seems this comment was copy and paste from another method.

Reply:
Fixed!